NuRadioReco/modules/trigger/envelopeTrigger.py

```diff
@@ -0,0 +1,161 @@
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units
+from NuRadioReco.modules.trigger.highLowThreshold import get_majority_logic
+from NuRadioReco.framework.trigger import EnvelopeTrigger
+import NuRadioReco.utilities.fft
+import numpy as np
+import scipy.signal
+import copy
+import time
+import logging
+
+logger = logging.getLogger('envelopeTrigger')
+
+
+def get_envelope_triggers(trace, threshold):  # define trigger constraint for each channel
+
+    """
+    calculates a Hilbert-envelope based trigger
+
+    Parameters
+    ----------
+    trace: array of floats
+        the signal trace
+    threshold: float
+        the threshold
+    Returns
+    -------
+    triggered bins: array of bools
+        the bins where the trigger condition is satisfied
+    """
+
+    return np.abs(scipy.signal.hilbert(trace)) > threshold
+
+
+class triggerSimulator:
+    """
+    Calculate a simple amplitude trigger depending on the Hilbert-envelope.
+    """
+
+    def __init__(self):
+        self.__t = 0
+        self.begin()
+
+    def begin(self, debug=False):
+        self.__debug = debug
+
+
+    @register_run()
+    def run(self, evt, station, det, passband, order, threshold, coinc_window, number_coincidences=2, triggered_channels=None,
+             trigger_name='envelope_trigger'):
+        """
+        simulate simple trigger logic, no time window, just threshold in all channels
```

Comment:
Doumentation looks outdated. There should be time windows, correct? Please update.

Reply:
Done.